### PR TITLE
verify vo / support center contacts (SOFTWARE-3329)

### PR DESCRIPTION
**Pull request by edquist**
"Based" off #667, except rewritten to match the style of the other tests.

The bigger the functions, the harder they fall?  Er, i mean, the more intimidating they are to read.

### CODE REVIEW

**Comment by edquist** [comment]
Balls there's a lot of contact ID errors.

```
$ ./src/tests/verify_resources.py

*** Resources must have an Administrative Contact

In 'Lawrence Berkley National Laboratory/LBL_HPCS/LBL_HPCS.yaml', Resource 'LBL_HPCS' has no Administrative Contact

In 'Oak Ridge National Laboratory/ORNL/ORNL.yaml', Resource 'ORNL_ALICE' has no Administrative Contact

In 'University of California San Diego/UCSD CMS Tier2/UCSD CMS Glideins.yaml', Resource 'UCSD_glidein_submit_4' has no Administrative Contact

In 'University of California San Diego/UCSD CMS Tier2/UCSDT2-ITB.yaml', Resource 'UCSDT2-ITB1' has no Administrative Contact

In 'University of California San Diego/UCSD OSG/UCSD glidein.yaml', Resource 'UCSD glidein' has no Administrative Contact

*** Resources must have a Security Contact

In 'Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml', Resource 'USCMS-FNAL-WC1-OPP1' has no Security Contact

In 'Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml', Resource 'USCMS-FNAL-WC1-SE' has no Security Contact

In 'Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml', Resource 'USCMS-FNAL-WC1-SE2' has no Security Contact

In 'Fermi National Accelerator Laboratory/FermiGrid/GPGRID.yaml', Resource 'FNAL_GPGRID_CE_03' has no Security Contact

In 'Fermi National Accelerator Laboratory/FermiGrid/GPGRID.yaml', Resource 'FNAL_GPGRID_CE_04' has no Security Contact

In 'Fermi National Accelerator Laboratory/FermiGrid/GPGRID.yaml', Resource 'FNAL_JOBSUB_01' has no Security Contact

In 'Fermi National Accelerator Laboratory/FermiGrid/GPGRID.yaml', Resource 'FNAL_JOBSUB_02' has no Security Contact

In 'University of California San Diego/UCSD CMS Tier2/UCSDT2-ITB.yaml', Resource 'UCSDT2-ITB1' has no Security Contact

In 'University of California San Diego/UCSD CMS Tier2/UCSDT2-ITB.yaml', Resource 'UCSDT2-ITB2' has no Security Contact

In 'University of Chicago/UChicago/UChicagoCachingInfrastructure.yaml', Resource 'OSGCONNECT_ORIGIN' has no Security Contact

In 'University of Georgia/UGA Research Computing Center/UGA_SGrid.yaml', Resource 'UGA_Sgrid_CE' has no Security Contact

*** Contact IDs must exist in contact repo

Support Center 'Beocat Support' has unknown 'Miscellaneous Contact' Contact ID '74c85e7fd273bf612adf1434a97f8eb123517698' (Beocat Support)

Support Center 'Beocat Support' has unknown 'Notifications Contact' Contact ID '74c85e7fd273bf612adf1434a97f8eb123517698' (Beocat Support)

Support Center 'Beocat Support' has unknown 'Operations Contact (Ticketing)' Contact ID '74c85e7fd273bf612adf1434a97f8eb123517698' (Beocat Support)

Support Center 'Beocat Support' has unknown 'Security Contact' Contact ID '74c85e7fd273bf612adf1434a97f8eb123517698' (Beocat Support)

Support Center 'CCSUP' has unknown 'Notifications Contact' Contact ID '9847f3cc5d27dd87b137b71cdcb6918a816c9194' (Roy Chartier)

Support Center 'CCSUP' has unknown 'Operations Contact (Ticketing)' Contact ID '9847f3cc5d27dd87b137b71cdcb6918a816c9194' (Roy Chartier)

Support Center 'CCSUP' has unknown 'Security Contact' Contact ID '9847f3cc5d27dd87b137b71cdcb6918a816c9194' (Roy Chartier)

Support Center 'CERN' has unknown 'Notifications Contact' Contact ID 'f3e62bda9a34084a698158d158fdaeef7dc1b7e5' (Rob Quick)

Support Center 'CERN' has unknown 'Notifications Contact' Contact ID '17ec5a8056c96b24934517aa31eac84b0e98608b' (Maria Dimou)

Support Center 'CERN' has unknown 'Operations Contact (Ticketing)' Contact ID 'f3e62bda9a34084a698158d158fdaeef7dc1b7e5' (Rob Quick)

Support Center 'CERN' has unknown 'Operations Contact (Ticketing)' Contact ID '17ec5a8056c96b24934517aa31eac84b0e98608b' (Maria Dimou)

Support Center 'CERN' has unknown 'Security Contact' Contact ID 'f3e62bda9a34084a698158d158fdaeef7dc1b7e5' (Rob Quick)

Support Center 'Community Support Center' has unknown 'Notifications Contact' Contact ID 'a86faef88b867839882fcc12d719171113acafd6' (OSG Operations Center)

Support Center 'Community Support Center' has unknown 'Operations Contact (Ticketing)' Contact ID 'a86faef88b867839882fcc12d719171113acafd6' (OSG Operations Center)

Support Center 'Community Support Center' has unknown 'Security Contact' Contact ID 'a86faef88b867839882fcc12d719171113acafd6' (OSG Operations Center)

Support Center 'DOSAR' has unknown 'Operations Contact (Ticketing)' Contact ID 'b4be8384e7611b25431440bd53e97ee8bb699f99' (DOSAR Support (Patrick McGuigan))

Support Center 'DOSAR' has unknown 'Security Contact' Contact ID 'b4be8384e7611b25431440bd53e97ee8bb699f99' (DOSAR Support (Patrick McGuigan))

Support Center 'Engagement' has unknown 'Miscellaneous Contact' Contact ID '55782e9da2fa03176ca211d4a59bd8b39d3f020f' (Engage Team)

Support Center 'Engagement' has unknown 'Notifications Contact' Contact ID '55782e9da2fa03176ca211d4a59bd8b39d3f020f' (Engage Team)

Support Center 'Engagement' has unknown 'Operations Contact (Ticketing)' Contact ID '55782e9da2fa03176ca211d4a59bd8b39d3f020f' (Engage Team)

Support Center 'Engagement' has unknown 'Security Contact' Contact ID '55782e9da2fa03176ca211d4a59bd8b39d3f020f' (Engage Team)

Support Center 'Fermilab' has unknown 'Notifications Contact' Contact ID '9ee3c5986e26fcbf82a16ea945d7aa98a612938e' (Fermilab Service Desk)

Support Center 'Fermilab' has unknown 'Operations Contact (Ticketing)' Contact ID '9ee3c5986e26fcbf82a16ea945d7aa98a612938e' (Fermilab Service Desk)

Support Center 'Fermilab' has unknown 'Security Contact' Contact ID '89674c61dffeda0d60cf0519b71cecc80de9905d' (Fermilab Security Group)

Support Center 'Fermilab_FIFE_Offsite' has unknown 'Miscellaneous Contact' Contact ID '51a1ed565309ea333bc008d09a1f34088f9a65e3' (Michael Kirby)

Support Center 'Fermilab_FIFE_Offsite' has unknown 'Notifications Contact' Contact ID '9ee3c5986e26fcbf82a16ea945d7aa98a612938e' (Fermilab Service Desk)

Support Center 'Fermilab_FIFE_Offsite' has unknown 'Operations Contact (Ticketing)' Contact ID '51a1ed565309ea333bc008d09a1f34088f9a65e3' (Michael Kirby)

Support Center 'Fermilab_FIFE_Offsite' has unknown 'Security Contact' Contact ID '89674c61dffeda0d60cf0519b71cecc80de9905d' (Fermilab Security Group)

Support Center 'Fermilab_FIFE_Offsite' has unknown 'Security Contact' Contact ID 'a6821a9abd2bad3c68f6d0301eca6fd52f960f35' (FermiGrid Security)

Support Center 'GGUS' has unknown 'Miscellaneous Contact' Contact ID 'f3e62bda9a34084a698158d158fdaeef7dc1b7e5' (Rob Quick)

Support Center 'GGUS' has unknown 'Notifications Contact' Contact ID 'a86faef88b867839882fcc12d719171113acafd6' (OSG Operations Center)

Support Center 'GGUS' has unknown 'Operations Contact (Ticketing)' Contact ID '0ccf183520801f2a51ccdd2b658d179b9b968c41' (GGUS Ticket Exchange)

Support Center 'GGUS' has unknown 'Security Contact' Contact ID '2e928efabe1bf2076536fc133ea6015a4a24646d' (EGEE Security (GGUS - Romain Wartel))

Support Center 'GPN' has unknown 'Notifications Contact' Contact ID 'ff09dcb661c50859b475eb65071408c8f53c5c35' (GPN Operations Contact)

Support Center 'GPN' has unknown 'Operations Contact (Ticketing)' Contact ID 'ff09dcb661c50859b475eb65071408c8f53c5c35' (GPN Operations Contact)

Support Center 'GridUNESP' has unknown 'Miscellaneous Contact' Contact ID '417b440ef784bfd64f5261e852881dce8ff5da51' (GridUNESP Administrators)

Support Center 'GridUNESP' has unknown 'Miscellaneous Contact' Contact ID '6eb5e83a829d77c97b6406cd77b8546826e74dd6' (Angelo  de Souza Santos)

Support Center 'GridUNESP' has unknown 'Notifications Contact' Contact ID '417b440ef784bfd64f5261e852881dce8ff5da51' (GridUNESP Administrators)

Support Center 'GridUNESP' has unknown 'Notifications Contact' Contact ID '6eb5e83a829d77c97b6406cd77b8546826e74dd6' (Angelo  de Souza Santos)

Support Center 'GridUNESP' has unknown 'Operations Contact (Ticketing)' Contact ID '417b440ef784bfd64f5261e852881dce8ff5da51' (GridUNESP Administrators)

Support Center 'GridUNESP' has unknown 'Operations Contact (Ticketing)' Contact ID '6eb5e83a829d77c97b6406cd77b8546826e74dd6' (Angelo  de Souza Santos)

Support Center 'GridUNESP' has unknown 'Security Contact' Contact ID '417b440ef784bfd64f5261e852881dce8ff5da51' (GridUNESP Administrators)

Support Center 'LBNL-DSD-support' has unknown 'Notifications Contact' Contact ID 'f3e52e6e61abc55a928031a2c843d7e2eb67728c' (Doug Olson)

Support Center 'LBNL-DSD-support' has unknown 'Operations Contact (Ticketing)' Contact ID 'f3e52e6e61abc55a928031a2c843d7e2eb67728c' (Doug Olson)

Support Center 'LBNL-DSD-support' has unknown 'Security Contact' Contact ID 'f3e52e6e61abc55a928031a2c843d7e2eb67728c' (Doug Olson)

Support Center 'LC hotline' has unknown 'Notifications Contact' Contact ID 'c2cb7234bc2f0f824e3be4c5bd4daaac9408ff1a' (Jeff Cunningham)

Support Center 'LC hotline' has unknown 'Operations Contact (Ticketing)' Contact ID 'c2cb7234bc2f0f824e3be4c5bd4daaac9408ff1a' (Jeff Cunningham)

Support Center 'LC hotline' has unknown 'Security Contact' Contact ID 'c2cb7234bc2f0f824e3be4c5bd4daaac9408ff1a' (Jeff Cunningham)

Support Center 'LHCONE_SC' has unknown 'Miscellaneous Contact' Contact ID 'f3e62bda9a34084a698158d158fdaeef7dc1b7e5' (Rob Quick)

Support Center 'LHCONE_SC' has unknown 'Notifications Contact' Contact ID '3cdd93ac465eb6602055e2ed60794e84f1c63a44' (WLCG perfSONAR Support)

Support Center 'LIGO' has unknown 'Miscellaneous Contact' Contact ID 'b64fc0ac7c5029072fe71ce5a3c5fdf6a130cdbf' (Jeff Minelli)

Support Center 'LIGO' has unknown 'Notifications Contact' Contact ID 'bc9da1ffac6dad2baa1d6766c3230ad2ae85fa57' (Brian Moe)

Support Center 'LIGO' has unknown 'Operations Contact (Ticketing)' Contact ID '93ddba0774c057f5368c146e528ad3e94ac16755' (LIGO Help Desk)

Support Center 'LIGO' has unknown 'Operations Contact (Ticketing)' Contact ID 'bc9da1ffac6dad2baa1d6766c3230ad2ae85fa57' (Brian Moe)

Support Center 'LIGO' has unknown 'Security Contact' Contact ID 'bc9da1ffac6dad2baa1d6766c3230ad2ae85fa57' (Brian Moe)

Support Center 'LSST' has unknown 'Miscellaneous Contact' Contact ID 'b580be4fd0c8bf220e6aaa90f81eb43bf4081ac2' (Jeffrey Kantor)

Support Center 'LSST' has unknown 'Miscellaneous Contact' Contact ID '2acb0ea0f6754fd8498c6436cee29cdca1b4addb' (John Peterson)

Support Center 'LSST' has unknown 'Notifications Contact' Contact ID 'a02fd4ae3db01bd79f6b880009b70042aa478a00' (LSST VO support)

Support Center 'LSST' has unknown 'Operations Contact (Ticketing)' Contact ID 'a02fd4ae3db01bd79f6b880009b70042aa478a00' (LSST VO support)

Support Center 'LSST' has unknown 'Security Contact' Contact ID 'a02fd4ae3db01bd79f6b880009b70042aa478a00' (LSST VO support)

Support Center 'NERSC' has unknown 'Notifications Contact' Contact ID '784a271b0eb4af191a240497540b61c60dc1ec17' (Eric Hjort)

Support Center 'NERSC' has unknown 'Operations Contact (Ticketing)' Contact ID '63c80b0ecd5f7fac5bd61f4507a9dd2f9a48b1a8' (NERSC Grid Trouble (Shreyas Cholia))

Support Center 'NERSC' has unknown 'Security Contact' Contact ID '18837099f4a4e66d76fae5230eeec9684793435e' (NERSC Security Group (Steve Chan))

Support Center 'NWICG' has unknown 'Miscellaneous Contact' Contact ID 'a8c93626f1c3ddf41151407df5474a2a2637ce04' (Kevin Colby)

Support Center 'NWICG' has unknown 'Notifications Contact' Contact ID 'ac919830e01b5fe839e794bdf32a4a419f802fb2' (NWICG Support)

Support Center 'NWICG' has unknown 'Operations Contact (Ticketing)' Contact ID 'ac919830e01b5fe839e794bdf32a4a419f802fb2' (NWICG Support)

Support Center 'NWICG' has unknown 'Security Contact' Contact ID '38fe4083bc4193ea1e74571957d667bca08b9234' (NWICG Security)

Support Center 'NYSGRID' has unknown 'Miscellaneous Contact' Contact ID '5aafc9af56cc8789093c310c75a9564d2c72caf6' (Jon Bednasz)

Support Center 'NYSGRID' has unknown 'Miscellaneous Contact' Contact ID '7ced1836cd81f6c556ec6a02e55d41775ddeab57' (Steve Gallo)

Support Center 'NYSGRID' has unknown 'Notifications Contact' Contact ID '5aafc9af56cc8789093c310c75a9564d2c72caf6' (Jon Bednasz)

Support Center 'NYSGRID' has unknown 'Notifications Contact' Contact ID '7ced1836cd81f6c556ec6a02e55d41775ddeab57' (Steve Gallo)

Support Center 'NYSGRID' has unknown 'Operations Contact (Ticketing)' Contact ID '5aafc9af56cc8789093c310c75a9564d2c72caf6' (Jon Bednasz)

Support Center 'NYSGRID' has unknown 'Operations Contact (Ticketing)' Contact ID '7ced1836cd81f6c556ec6a02e55d41775ddeab57' (Steve Gallo)

Support Center 'NYSGRID' has unknown 'Security Contact' Contact ID '5aafc9af56cc8789093c310c75a9564d2c72caf6' (Jon Bednasz)

Support Center 'NYSGRID' has unknown 'Security Contact' Contact ID '7ced1836cd81f6c556ec6a02e55d41775ddeab57' (Steve Gallo)

Support Center 'SBGrid' has unknown 'Miscellaneous Contact' Contact ID '0598213e60c06f6721be660a9c8891c7e64c3969' (SBGrid Helpdesk)

Support Center 'SBGrid' has unknown 'Miscellaneous Contact' Contact ID '748143b943af988693da79cd03dc27def18c6cd6' (Jason Key)

Support Center 'SBGrid' has unknown 'Notifications Contact' Contact ID '748143b943af988693da79cd03dc27def18c6cd6' (Jason Key)

Support Center 'SBGrid' has unknown 'Operations Contact (Ticketing)' Contact ID '1777384243c590f66f0ef6b11c93477c80e3062d' (SBGrid Operations)

Support Center 'SBGrid' has unknown 'Operations Contact (Ticketing)' Contact ID '748143b943af988693da79cd03dc27def18c6cd6' (Jason Key)

Support Center 'SBGrid' has unknown 'Security Contact' Contact ID '748143b943af988693da79cd03dc27def18c6cd6' (Jason Key)

Support Center 'STAR' has unknown 'Notifications Contact' Contact ID 'afc36b3168f9f7784a113cf55ab552fc772862a7' (STAR Grid)

Support Center 'STAR' has unknown 'Operations Contact (Ticketing)' Contact ID 'afc36b3168f9f7784a113cf55ab552fc772862a7' (STAR Grid)

Support Center 'STAR' has unknown 'Security Contact' Contact ID 'afc36b3168f9f7784a113cf55ab552fc772862a7' (STAR Grid)

Support Center 'SUPERB-SC' has unknown 'Miscellaneous Contact' Contact ID 'e640880f700d66f2b5a945e7810e22310412e90b' (Armando Fella)

Support Center 'SUPERB-SC' has unknown 'Notifications Contact' Contact ID 'e640880f700d66f2b5a945e7810e22310412e90b' (Armando Fella)

Support Center 'SUPERB-SC' has unknown 'Operations Contact (Ticketing)' Contact ID 'e640880f700d66f2b5a945e7810e22310412e90b' (Armando Fella)

Support Center 'SUPERB-SC' has unknown 'Security Contact' Contact ID 'e640880f700d66f2b5a945e7810e22310412e90b' (Armando Fella)

Support Center 'SURAgrid GOC' has unknown 'Notifications Contact' Contact ID '25a99de8af1d270b6cae832961bbccb316c8c23b' (SURAgrid VO Notifications)

Support Center 'SURAgrid GOC' has unknown 'Operations Contact (Ticketing)' Contact ID '60f3acf5c35cb1515e326f70de33d7821bbae284' (SURAgrid VO Support)

Support Center 'SURAgrid GOC' has unknown 'Security Contact' Contact ID '2540192a26d757da3096dfa2e4ac0e10fb1bd9a2' (SURAgrid VO Security)

Support Center 'TIGRE' has unknown 'Notifications Contact' Contact ID '33f0d76f176086e323ca26e1cc04f9d70f78297f' (Ravi Vadapalli)

Support Center 'TIGRE' has unknown 'Operations Contact (Ticketing)' Contact ID '33f0d76f176086e323ca26e1cc04f9d70f78297f' (Ravi Vadapalli)

Support Center 'TIGRE' has unknown 'Security Contact' Contact ID '33f0d76f176086e323ca26e1cc04f9d70f78297f' (Ravi Vadapalli)

Support Center 'UC CI' has unknown 'Notifications Contact' Contact ID 'ef7276d677048574b1cd6c8bececb2737428a374' (CI VO Support Center (Greg Cross))

Support Center 'UC CI' has unknown 'Operations Contact (Ticketing)' Contact ID 'ca1a3208208d3ab9012b26bc31888d440bf79679' (UC Teraport Support (Greg Cross))

Support Center 'UC CI' has unknown 'Security Contact' Contact ID 'ef7276d677048574b1cd6c8bececb2737428a374' (CI VO Support Center (Greg Cross))

Support Center 'UCSD OSG' has unknown 'Notifications Contact' Contact ID 'a599ec0e9ceabbe0267e5d5ca7efd228ae3b3958' (UCSD T2 Support Group)

Support Center 'UCSD OSG' has unknown 'Operations Contact (Ticketing)' Contact ID 'a599ec0e9ceabbe0267e5d5ca7efd228ae3b3958' (UCSD T2 Support Group)

Support Center 'UCSD OSG' has unknown 'Security Contact' Contact ID 'a599ec0e9ceabbe0267e5d5ca7efd228ae3b3958' (UCSD T2 Support Group)

Support Center 'USATLAS' has unknown 'Notifications Contact' Contact ID 'a0652278a29ec36b78a1e0c6d4a1b199a8fa0ed0' (US ATLAS Support (Danong Yu))

Support Center 'USATLAS' has unknown 'Operations Contact (Ticketing)' Contact ID '9e84ced8a2a2d261b3543ca29dae5861ea6be172' (USATLAS RT Ticketing System)

Support Center 'USATLAS' has unknown 'Security Contact' Contact ID '0a09bf33c39b4ee3b4e6f2065b5eb849fa5750c9' (Michael Ernst)

Support Center 'USATLAS_Tier3' has unknown 'Miscellaneous Contact' Contact ID '7a88370282be73e3756393b891af447075e0a9e8' (Rik Yoshida)

Support Center 'USATLAS_Tier3' has unknown 'Notifications Contact' Contact ID '7a88370282be73e3756393b891af447075e0a9e8' (Rik Yoshida)

Support Center 'USATLAS_Tier3' has unknown 'Operations Contact (Ticketing)' Contact ID '9e84ced8a2a2d261b3543ca29dae5861ea6be172' (USATLAS RT Ticketing System)

Support Center 'USCMS' has unknown 'Notifications Contact' Contact ID '6b51e56250a10aa1e00cd14e5f909f06f06f154d' (USCMS Operations)

Support Center 'USCMS' has unknown 'Operations Contact (Ticketing)' Contact ID '5d196fa895c59db274f1e3f71ddb709a67a8520b' (Footprints Remedy (FNAL) Ticket Exchange)

Support Center 'USCMS' has unknown 'Security Contact' Contact ID '89674c61dffeda0d60cf0519b71cecc80de9905d' (Fermilab Security Group)

Support Center 'USCMS_Tier2' has unknown 'Notifications Contact' Contact ID '6b51e56250a10aa1e00cd14e5f909f06f06f154d' (USCMS Operations)

Support Center 'USCMS_Tier2' has unknown 'Security Contact' Contact ID '89674c61dffeda0d60cf0519b71cecc80de9905d' (Fermilab Security Group)

Support Center 'USCMS_Tier3' has unknown 'Notifications Contact' Contact ID 'cc5c08227fa6163d6c8bc487dc9d76cbe45a89f3' (USCMS Tier 3 Support)

Support Center 'USCMS_Tier3' has unknown 'Operations Contact (Ticketing)' Contact ID 'cc5c08227fa6163d6c8bc487dc9d76cbe45a89f3' (USCMS Tier 3 Support)

Support Center 'USCMS_Tier3' has unknown 'Security Contact' Contact ID '89674c61dffeda0d60cf0519b71cecc80de9905d' (Fermilab Security Group)

Support Center 'UW-IT' has unknown 'Miscellaneous Contact' Contact ID '43155ca51189c1e2f907e51ec2d00fb4bacd95ee' (Chance Reschke)

Support Center 'UW-IT' has unknown 'Miscellaneous Contact' Contact ID '5dd3d36a69420319b340f98095c488c2828190ac' (David Cox)

Support Center 'UW-IT' has unknown 'Notifications Contact' Contact ID '2b2b1ec928b263238527e1d0ca78dd2585a67162' (Carlos Griffin)

Support Center 'fGOC' has unknown 'Notifications Contact' Contact ID 'a04826e20031fb30b5102cd7cf0efb9dd4a29c53' (Florida fGOC (Jorge Rodriguez))

Support Center 'fGOC' has unknown 'Operations Contact (Ticketing)' Contact ID 'a04826e20031fb30b5102cd7cf0efb9dd4a29c53' (Florida fGOC (Jorge Rodriguez))

Support Center 'fGOC' has unknown 'Security Contact' Contact ID 'a04826e20031fb30b5102cd7cf0efb9dd4a29c53' (Florida fGOC (Jorge Rodriguez))

Support Center 'osg-glidein-factory' has unknown 'Notifications Contact' Contact ID '41ae6f74463c2e689725a09ec016174c23b554f6' (Glide-In WMS)

Support Center 'osg-glidein-factory' has unknown 'Operations Contact (Ticketing)' Contact ID '41ae6f74463c2e689725a09ec016174c23b554f6' (Glide-In WMS)

Support Center 'osg-glidein-factory' has unknown 'Security Contact' Contact ID '27e128b4e4daa714b2bfbf925167800dc42f8e41' (Glide-In WMS Security)

*** Contact names must match in contact repo

Support Center 'USALICE': 'Security Contact' Contact ID 'c0b37d66f9d39c4a37a83d20dfaf97f80d0c197e' (Pete Eby) does not match name in contact repo (Peter Eby)

Support Center 'USATLAS': 'Notifications Contact' Contact ID '1e47726bf41f2128059de31cf7bc9c8d5dcadad6' (Xin Zhao) does not match name in contact repo (BNL Atlas Tier 1 support)

Support Center 'USATLAS': 'Operations Contact (Ticketing)' Contact ID '1e47726bf41f2128059de31cf7bc9c8d5dcadad6' (Xin Zhao) does not match name in contact repo (BNL Atlas Tier 1 support)

221 Resource Group files processed.

126 error(s) encountered.

The command "./src/tests/verify_resources.py" exited with 1.
```

**Comment by edquist** [comment]
Per @matyasselmeci, I've dropped all the unknown contacts (ie, those not in the contacts repo) from support centers.  Also i corrected the few contact name mismatches. This resulted in a passing build / with no more errors (d682c21).

Also per Mat in SOFTWARE-3329, i have removed the support center contact checks, as we apparently eventually want to stop tracking support centers altogether.

File: src/tests/verify_resources.py [hunk 1]
```diff
@@ -95,11 +104,19 @@ def main():
     warnings += test_10_res_admin_contact(rgs, rgfns)
     warnings += test_11_res_sec_contact(rgs, rgfns)
     errors += test_12_res_contact_id_fmt(rgs, rgfns)
+    errors += test_12_vo_contact_id_fmt(vos, vofns)
+    # per SOFTWARE-3329, we are not checking support center contacts
+#   errors += test_12_sc_contact_id_fmt(support_centers)
     errors += test_13_res_contacts_exist(rgs, rgfns, contacts)
+    errors += test_13_vo_contacts_exist(vos, vofns, contacts)
+#   errors += test_13_sc_contacts_exist(support_centers, contacts)
```
Comment (brianhlin) [inline src/tests/verify_resources.py:112]:
```suggestion
    # per SOFTWARE-3329, we are not checking support center contacts
#   errors += test_13_sc_contacts_exist(support_centers, contacts)
```

File: src/tests/verify_resources.py [hunk 2]
```diff
@@ -95,11 +104,19 @@ def main():
...
     errors += test_14_res_contacts_match(rgs, rgfns, contacts)
+    errors += test_14_vo_contacts_match(vos, vofns, contacts)
+#   errors += test_14_sc_contacts_match(support_centers, contacts)
```
Comment (brianhlin) [inline src/tests/verify_resources.py:115]:
```suggestion
    # per SOFTWARE-3329, we are not checking support center contacts
#   errors += test_14_sc_contacts_match(support_centers, contacts)
```

**Pull request merged**